datasets/ng/nigsac_sanctions/ng_nigsac_sanctions.yml

```diff
@@ -48,6 +48,11 @@ data:
   format: HTML
 dates:
   formats: ["%m/%d/%Y %H:%M:%S %p"]
+http:
+  additional_retry_statuses: [443]
```

Comment:
Why 443? Isn't this just the HTTPS port? And why POST, isn't it failing on the GET right now?